netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java

```diff
@@ -565,18 +565,6 @@ private AsciiString as(String string) {
     return new AsciiString(string);
   }
 
-  private void mockContextForPing() {
```

Comment:
Weird ... I swear I deleted this in another PR.  Possibly one I haven't submitted yet?